dao/common.go

```diff
@@ -0,0 +1,59 @@
+package dao
+
+import (
+	"fmt"
+
+	m "github.com/RedHatInsights/sources-api-go/model"
+)
+
+func GetFrom(resourceType string) (*m.EventModelDao, error) {
```

Comment:
Can we rename this to something more descriptive? `GetFrom` is a bit too vague - maybe `GetDaoFromType` or something along those lines

Reply:
to be honest in addition I was thinking how to remove "Get" part as it looks to me redundant but then "Dao" would became also redundant :)  :
`dao.DaoFrom("Source")` :) 

Descriptive part is here also first parameter - so it should be understandable enough - anyway I change it as well for certitude to `GetFromResourceType`